apps/explorer/src/utils/growthbook.ts

```diff
@@ -29,4 +29,5 @@ export async function loadFeatures() {
 
 export enum GROWTHBOOK_FEATURES {
     USE_TEST_NET_ENDPOINT = 'testnet-selection',
+    MODULE_VIEW_INVOKE_FUNCTIONS = 'explorer-module-view-invoke-functions',
```

Comment:
I love that we feature flagged this, makes it so much easier to land this incrementally!